packages/ui/src/components/va-list/VaList.vue

```diff
@@ -30,7 +30,6 @@ export default defineComponent({
   @import "variables";
 
   .va-list {
-    padding: var(--va-list-padding);
```

Comment:
This change might break styles for end users.

Reply:
@rustem-nasyrov, should we care about it, if we are going to release 1.5.0?

Reply:
@m0ksem, I think for now we don't, but I'm wondering is users will notice it or not. :)